### PR TITLE
Modernize Python 2 code to get ready for Python 3

### PR DESCRIPTION
**Pull request by cclauss**
Make the minimal, safe changes required to convert the repo's code to be syntax compatible with both Python 2 and Python 3. More work will be required to complete a port to Python 3 but this is a minimal, safe first step.

* [futurize stage 1](http://python-future.org/futurize_cheatsheet.html#step-1-modern-py2-code)
* print() function
* from __future__ import absolute_import
* octal numbers (0077 --> 0o077)
* new style exceptions

Signed-off-by: cclauss <cclauss@bluewin.ch>

### CODE REVIEW

File: crawler/config_parser.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
+from __future__ import absolute_import
 import logging
 
 from configobj import ConfigObj
 from validate import Validator
 
-from utils import misc
+from .utils import misc
```
Comment (sahilsuneja1) [inline crawler/config_parser.py]: These dots, I don't think we need them. Do we?

Comment (sahilsuneja1) [reply crawler/config_parser.py]: Removing these might also fix the failing style tests at travis

Comment (cclauss) [reply crawler/config_parser.py]: In Python 2 these dots are never needed but in Python 3 they might be for intrapackage imports.  https://www.python.org/dev/peps/pep-0328/

Comment (sahilsuneja1) [reply crawler/config_parser.py]: Alright, can you please fix the travis errors then.

**Pull request closed without merging**